services/alertmanager/alertmanager_test.go

```diff
@@ -36,3 +43,21 @@ func TestAlertmanager(t *testing.T) {
 
 	require.NoError(t, svc.IsReady(context.Background()))
 }
+
+func TestCollect(t *testing.T) {
+	err := os.Setenv("PERCONA_TEST_SHIPPED_RULE_TEMPLATE_PATH", testShippedFilePath)
+	require.NoError(t, err)
+	err = os.Setenv("PERCONA_TEST_USER_DEFINED_RULE_TEMPLATE_PATH", testUserDefinedFilePath)
+	require.NoError(t, err)
```

Comment:
That can affect other tests that run parallel, probably better just set struct fields directly.